api/collections/serializers.py

```diff
@@ -152,7 +152,7 @@ class Meta:
         'subjects',
         'status',
     ])
-    id = IDField(source='_id', read_only=True)
+    id = IDField(source='guid._id', read_only=True)
```

Comment:
Will this result in a `JOIN` for every serialized "Collected Meta"? Can you verify that this isn't causing a large performance penalty? 

The Django Debug Toolbar is useful for this.

Reply:
(# of search results, # of queries, time (ms))

with <guid._id>: 	(1, 20, 44.83), (2, 37, 69.53), (3, 54, 107), (4, 71, 143.33)
with <_id>:	(1, 19, 39.65), (2, 35, 60.75), (3, 51, 99.65), (4, 67, 134)

Seems with id=<guid._id> we are making 1 additional query per search result compared to same scenario with id=<_id>.